test/test_release.py

```diff
@@ -1,24 +1,24 @@
 #! /usr/bin/env python
 
+import filecmp
 import os
 import unittest
-import yaml
-import filecmp
 import urllib.request
 from tempfile import mkstemp
-from github import Github, GithubException
 from zipfile import ZipFile
 
+import yaml
+from github import Github, GithubException
 from pytransifex.exceptions import PyTransifexException
 
-from qgispluginci.parameters import Parameters, DASH_WARNING
+from qgispluginci.exceptions import GithubReleaseNotFound
+from qgispluginci.parameters import DASH_WARNING, Parameters
 from qgispluginci.release import release
 from qgispluginci.translation import Translation
-from qgispluginci.exceptions import GithubReleaseNotFound
 from qgispluginci.utils import replace_in_file
 
 # if change, also update on .travis.yml and CHANGELOG.md
-RELEASE_VERSION_TEST = "0.1.2"
+RELEASE_VERSION_TEST = "1.8.3"
```

Comment:
Tests are failing because this line has changed.
And the changelog is also tested in the release process : https://github.com/opengisch/qgis-plugin-ci/blob/master/test/test_release.py#L135..L152

Reply:
and mostly the release is needed to play with its assets
https://github.com/opengisch/qgis-plugin-ci/releases/tag/0.1.2

Reply:
> and mostly the release is needed to play with its assets
> https://github.com/opengisch/qgis-plugin-ci/releases/tag/0.1.2

This is what I was missing. So we need to point to the specific changelog created for testing (in test/fixtures).